packages/plugin-workflow-approval/src/client/user-interface/h5/component/ApprovalProcess.view.tsx

```
@@ -151,15 +136,15 @@ const getStepsResult = (result, t) => {
         (!(value.workflow != null && value.workflow.enabled) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)